src/test/scala/com/databricks/spark/xml/XmlSuite.scala

```diff
@@ -75,7 +75,9 @@ class XmlSuite extends FunSuite with BeforeAndAfterAll {
 
   override protected def beforeAll(): Unit = {
     super.beforeAll()
-    sqlContext = new SQLContext(new SparkContext("local[2]", "XmlSuite"))
+    val conf = new SparkConf().set("spark.sql.warehouse.dir",
+      Files.createTempDirectory("spark-warehouse").toString)
```

Comment:
ditto

Reply:
Here too. Let's leave some comments and make this as:

```
val conf = new SparkConf().set(
  "spark.sql.warehouse.dir",
  Files.createTempDirectory("spark-warehouse").toString)
```

Reply:
Fixed